### ISSUE TITLE
No fading. public void draw(Canvas canvas) never called 

### ISSUE THREAD

**Issue by ncapdevi** [square/picasso#1680]
I'm not sure how this is possible, my images are loading and have for awhile, but I recently noticed that they weren't fading.  I realized that the `PicassoDrawable` method `public void draw(Canvas canvas)` was where this would happen, and after putting breakpoints there, it seems that this method is never called. I'm not sure how this is possible, but any ideas?

**Comment by ncapdevi** [comment]
I believe the issues stems from me subclassing a [RoundedImageView](https://github.com/vinc3m1/RoundedImageView) and it somehow making the draw method not being called. This may not be a Picasso issue.

**Comment by GavinLi369** [comment]
Yes, This is the reason for your `RoundedImageView`. When `setImageDrawable(Drawable drawable)` method is invoked, the `RoundedImageView` converts the `PicassoDrawable` to `RoundedDrawable`. But the `RoundedDrawable` is only initialized by the `Bitmap` from the `PicassoDrawable`. So this leads to the loss of fading.

**Comment by ncapdevi** [comment]
@GavinLi369  That makes sense. Is there any easy work around to this?

**Comment by ncapdevi** [comment]
Still looking for a way to deal with this, any ideas?

**Comment by almaz010891** [comment]
Same issue
